templates/messages.properties.j2

```diff
@@ -22,7 +22,7 @@
 screen.welcome.welcome=Congratulations on bringing CAS online!  To learn how to authenticate, please review the default authentication handler configuration.
 screen.welcome.security={{ cas_screen_welcome_security }}
 screen.welcome.instructions={{ cas_screen_welcome_instructions }}
-screen.welcome.label.netid=<span class="accesskey">U</span>sername:
+screen.welcome.label.netid=<span class="accesskey">O</span>UNetID (4x4):
```

Comment:
don't hardcode values here, create a var that gets templated in, like cas_screen_welcome_instructions.  Note the naming convention of vars.